### PR TITLE
fix sudo not work expected

### LINKED ISSUE #1196: while sudo package not installed, precheck not work expected at 'create cluster'
### What is version of KubeKey has the issue?

v2.0.0

### What is your os environment?

debian 10

### KubeKey config file

_No response_

### A clear and concise description of what happend.

image

### Relevant log output

_No response_

### Additional information

_No response_

### PR DESCRIPTION
**Pull request by yinheli**
While sudo package not installed, precheck not work expected.

### What type of PR is this?
/kind bug

### Which issue(s) this PR fixes:

Fixes #1196

### Special notes for reviewers:

prepare test env:
```bash
apt remove sudo
```

### Does this PR introduced a user-facing change?

```release-note
None
```

### CODE REVIEW

File: pkg/bootstrap/precheck/tasks.go [hunk 1]
```diff
@@ -62,10 +70,7 @@ func (n *NodePreCheck) Execute(runtime connector.Runtime) error {
 		if err != nil || strings.Contains(res, "not found") {
 			results[software] = ""
 		} else {
-			if strings.HasPrefix(cmd, "which") {
```
Comment (24sama) [inline pkg/bootstrap/precheck/tasks.go:65]: Sorry, I don't know why should modify here. Here, the result of `docker` and `containerd` will show their version.

**Comment by 24sama** [comment]
Hi @yinheli 
Thanks for your contribution. I leave some comments, PTAL.

**Comment by 24sama** [comment]
Hi @yinheli Thanks again!
There is a last thing, please squash your commits, that will make the KubeKey's git commits history clearer.

**Comment by yinheli** [comment]
@24sama Got

**Comment by yinheli** [comment]
@24sama  Done, Thanks for your review.

**Comment by 24sama** [comment]
@all-contributors please add @yinheli for code

**Pull request merged**